### PR TITLE
missing 'return' and "libva" is available in the mingw(windows) environment.

### PR DESCRIPTION
**Pull request by testdrive-profiling-master**
missing 'return' code -> compiling error.
and
"libva" is available in the mingw(windows) environment.
image

### CODE REVIEW

**Comment by rkazants** [comment]
build_jenkins

**Comment by rkazants** [comment]
@vladimir-paramuzov, could you please review?

File: src/inference/include/openvino/runtime/intel_gpu/ocl/va.hpp [hunk 1]
```diff
@@ -12,8 +12,10 @@
 #pragma once
 
 #ifdef _WIN32
+#if !defined(__MINGW32__) && !defined(__MINGW64__)
 #    error "OpenCL VA-API interoperability is supported only on Linux-based platforms"
 #endif
```
Comment (rkazants) [inline src/inference/include/openvino/runtime/intel_gpu/ocl/va.hpp:17]:
```suggestion
#    if !defined(__MINGW32__) && !defined(__MINGW64__)
#        error "OpenCL VA-API interoperability is supported only on Linux-based platforms"
#    endif
```

**Comment by rkazants** [comment]
build_jenkins

**Comment by testdrive-profiling-master** [comment]
Author is changed correctly.~;

**Comment by rkazants** [comment]
build_jenkins

**Review by rkazants — APPROVED** [review]

**Review by p-durandin — APPROVED** [review]

**Pull request merged**